lib/travis/api/v3/services/build/build_prioritize.rb

```diff
@@ -0,0 +1,12 @@
+module Travis::API::V3
+  class Services::Build::BuildPrioritize < Service
+
+    def run
+      build = check_login_and_find(:build)
+      build_priority = build.owner.build_priority?
```

Comment:
@dmaity-celestialsys, owner may be a user as well, in this case we'll get undefined method error.